db/migrate/20170918160919_add_amount_paid_to_ticket_purchase.rb

```diff
@@ -0,0 +1,5 @@
+class AddAmountPaidToTicketPurchase < ActiveRecord::Migration
+  def change
+    add_column :ticket_purchases, :amount_paid, :float
```

Comment:
as said in https://github.com/openSUSE/osem/issues/1682#issuecomment-330082124 I would better add this in `ticket_purchase`